### DISCUSSION TITLE
What causes command output to appear in the IBM i Output view

### DISCUSSION THREAD

**Discussion by m-tyler** [codefori/vscode-ibmi#643] [IBM i]
I have seen commands run that produce a SPLF and that SPLF is sent to the IBM i Output view, but I have a vendor command (from Hawkeye Pathfinder) that does not.  Is there a special trick to this?  

The vendor command opens QSYS/QSYSPRT with defaults.

**Comment by worksofliam** [comment]
Interesting. I am not sure, I have always been under the impression that when you call a program that prints, it will write to standard out also (like all the compilers do!)

**Comment by m-tyler** [comment]
This is the server job output reports are tied to.  You can see a few QPRTLIBL entries, these tie to me using CL:DSPLIBL: SQL statement.  
![image](https://user-images.githubusercontent.com/1353665/164777641-1d3ccfec-8d90-413f-b39d-d719832b168e.png)

This is the second from bottom page and you can see more compile reports and other output.
![image](https://user-images.githubusercontent.com/1353665/164777746-6274da2f-98e7-4c63-b2fa-0ea62255fc46.png)

The entry PRTFCHKOUT is actually from our CMS product Rocket Aldon LM(i). So this vendor product does produce a report that gets picked up in the IBM i Output view.   

The Hawkeye tool produces a few messages to the joblog.
![image](https://user-images.githubusercontent.com/1353665/164778669-faeed814-b704-4882-9d88-97c378b648ac.png)


I was just hoping to be able to utilize the IBM I Output view for all functionality that does not require a 5250 screen.

**Comment by m-tyler** [comment]
My goal was to provide myself with a way to have a source search tool without the need for a filter and a source file.    DSPSRCSCN command from our cors reference vendor searches over what I want by providing a cross-referenced source search or a specific source file search or an *ALL/*ALL style of search.   These results would then be presented in the IBM i Output view.
